### DISCUSSION TITLE
Enable pagination for `{parents}` tag in relationships

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3193] [Feature Requests]
EE 7.2.12

I'm trying to develop a custom pagination for the `{parents}` tag, since there is no native pagination. 

I need to know how many absolute total results there are (when using `limit="10"`) to achieve it, but I can't: `{parents:total_results}` and `{parents:absolute_results}` are the same number.

**Comment by intoeetive** [comment]
@shbchk thank you for pushing EE to its limits :)

The reason why `absolute_results` is no different from `total_results` is because pagination is not supported :)

I checked and there's no "quick and easy" fix for us to enable that, so I am going to turn this into feature request

As a way around, you can use `{parents:entry_ids}` and pass it as parameter for `exp:channel:entries` tag in embedded template, and build pagination there

**Comment by shbchk** [comment]
Thanks for suggestion, I'll definitely give it a try :)
